README.md

```diff
@@ -116,12 +115,11 @@ const LinkedSheet = new Sheet(
 ...
 ```
 
-[View full source code](examples/FullExample.ts)
+[View full source code](examples/full-example/FullExample.ts)
```

Comment:
this should be `examples/workbooks/FullExample.ts`

Reply:
Good catch!